.gitignore

```diff
@@ -200,7 +200,9 @@ src/reso/agent/version.php
 src/reso/agent/reso
 src/www/ui/css/bootstrap/
 src/www/ui/scripts/bootstrap/
-
+CMakeCache.txt
+CMakeFiles/
+gen/
```

Comment:
These files should ideally be under `build/` folder. If they exist at ignored location, the CMake was not configured correctly by the developer. Thus, please remove them from `.gitignore`